README.md

```diff
@@ -7,7 +7,6 @@
   - [🛠 Built With ](#-built-with-)
```

Comment:
- [ ] Please the navigation and the logo should be algined on the same line as in the design 
- [ ] Please the color of the navigation links doesn't match the design the link is underlined which doesn't match the design 
- [ ] The profile icon doesn't match the one in the design there is  border around it in the design you should make sure it match the design
your design 
![image](https://github.com/jkanyi-web/bookstore-cra/assets/74744787/c4ebb087-ca4f-430a-84d6-398467edcb49)
Design 
![image](https://github.com/jkanyi-web/bookstore-cra/assets/74744787/321d8b8b-5280-40ca-950b-5cdc5220c5df)